### ISSUE TITLE
Plugin `:tmp_restart`

### ISSUE THREAD

**Issue by henrikbjorn** [socketry/falcon#223]
Anything like this Puma plugin for Falcon?

Most useful feature when developing :)

**Comment by ioquatix** [comment]
I agree with adding something like this, let me see what I can cook up.

**Comment by henrikbjorn** [comment]
@ioquatix Sorry to ping you like this. But I just saw this PR in my logs and was wondering if you thought of something? (Couldn't see it in code)
